docs/guidance/EnterpriseScaleAnalytics-NetworkArchitecture.md

```diff
@@ -9,7 +9,7 @@ Enterprise-Scale Analytics promises the possibility to easily share and access d
 
 ![Meshed Network Architecture](/docs/images/NetworkOptions-NetworkMesh.png)
 
-To explain the rationale behind the recommended design, this article will illustrate the advantages and disadvantages that come with each of the different network architecture approaches that were considered when designing Enterprise-Scale Analytics. Every design pattern will be evaluated along the following criteria: Cost, User Access Management, Service Management and Bandwidth. Each scenario will be analyzed with the following cross-Data Landing Zone use-case in mind:
+To explain the rationale behind the recommended design, this article will illustrate the advantages and disadvantages that come with each of the different network architecture approaches that were considered when designing Data Management & Analytics Scenario. Every design pattern will be evaluated along the following criteria: Cost, User Access Management, Service Management and Bandwidth. Each scenario will be analyzed with the following cross-Data Landing Zone use-case in mind:
```

Comment:
Rephrase? 

"This article explains the rationale behind this design recommendation for networking. With cross-Data Landing Zone interactions between services as the primary scenario, we evaluated the networking options across four key areas - Cost; User Access Management; Service Management and Bandwidth. "